extensions/amp-story-shopping/0.1/amp-story-shopping-config.js

```diff
@@ -16,12 +17,127 @@ import {
  */
 let ShoppingConfigDef;
 
+/**
+ * Validates string length of shopping config attributes
+ * @param {?string} field
+ * @param {?string} value
+ * @return {!Array<string>}
+ * @private
+ */
+function validateRequired(field, value) {
+  if (value == null) {
```

Comment:
Checking `== null` to verify that there is a missing field (`=== undefined`) is a bit weird. We should keep our strong typing, we mostly use strict equalities in our codebase.
Plz use the nullish operator `??` or `value === undefined`